README.md

```diff
@@ -79,36 +81,6 @@ juju controllers
 juju clouds
 ```
 
-### Deploy kubeflow
```

Comment:
Remove deploy kubeflow from here will make this document incomplete.

Suggest to keep it in both side or create a independent document for deploy kubeflow. It depends on we are creating two series or just one.